### PR TITLE
Disable MoralisClient option

### PR DESCRIPTION
**Pull request by KoenRijpstra**
- Added "Added Use Moralis 2.0 back-end integration and disable MoralisClient in Unity." option.

### CODE REVIEW

File: Runtime/Core/MoralisClient.cs [hunk 1]
```diff
@@ -21,6 +21,11 @@ public class MoralisClient : IDisposable
 
         public MoralisClient(ServerConnectionData connectionData, IWeb3Api web3Api = null, ISolanaApi solanaApi = null, IJsonSerializer jsonSerializer = null)
         {
+            if (MoralisSettings.MoralisData.DisableMoralisClient)
+            {
+                throw new ArgumentException("MoralisClient disabled in the settings. (I integrated Moralis 2.0 in my back-end and want to disable the client side connection.)");
```
Comment (xactant) [inline Runtime/Core/MoralisClient.cs:26]: Not a fan of this statement - should be simplified to something like: "MoralisClient disabled in settings. Integrate Moralis 2.0 with your backend services" or something similar.

**Review by dpradell-moralis — APPROVED** [review]
Reviewed. Not tested!

**Comment by dpradell-moralis** [comment]
Tested it using AuthenticationKit and custom backend and working smooth

File: Runtime/Core/MoralisClient.cs (hunk 1 above)
Comment (KoenRijpstra) [reply Runtime/Core/MoralisClient.cs]: Better?

**Pull request merged**